### PR TITLE
internal/website: add a secrets page

### PR DESCRIPTION
**Pull request by shantuo**

### CODE REVIEW

File: internal/website/content/pages/secrets.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: "Secrets"
+---
+
+Package secrets provides an easy and portable way to encrypt and decrypt
```
Comment (eliben) [inline internal/website/content/pages/secrets.md]: `secrets` (with backticks)

File: internal/website/content/pages/secrets.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+messages.
```
Comment (eliben) [inline internal/website/content/pages/secrets.md]: Can we add a bit more details here? From reading just this one line it's not clear why we need a portable cloud API for this as opposed to using Go's bultin crypto packages

File: internal/website/content/pages/secrets.md (hunk 1 above)
Comment (shantuo) [reply internal/website/content/pages/secrets.md]: Done.

File: internal/website/content/pages/secrets.md (hunk 2 above)
Comment (shantuo) [reply internal/website/content/pages/secrets.md]: Added some, PTAL.

**Pull request merged**